app/auto_users.go

```diff
@@ -48,12 +47,18 @@ func (a *App) CreateBasicUser(client *model.Client4) *model.AppError {
 		if resp.Error != nil {
 			return resp.Error
 		}
+
 		_, err := a.Srv.Store.User().VerifyEmail(ruser.Id, ruser.Email)
 		if err != nil {
 			return err
 		}
-		store.Must(a.Srv.Store.Team().SaveMember(&model.TeamMember{TeamId: basicteam.Id, UserId: ruser.Id}, *a.Config().TeamSettings.MaxUsersPerTeam))
+
+		_, err = a.Srv.Store.Team().SaveMember(&model.TeamMember{TeamId: basicteam.Id, UserId: ruser.Id}, *a.Config().TeamSettings.MaxUsersPerTeam)
```

Comment:
This is not exactly the same previous behavior, but I think it can be valid anyway.

Reply:
Alright, not changing this then @jespino